tests/test_migrations.py

```diff
@@ -0,0 +1,129 @@
+from cgi import test
+from hashlib import new
+from alembic.config import main as alembic_main
+import pytest
+import json
+
+from mds.config import DB_DSN, DEFAULT_AUTHZ_STR
+from mds.models import db
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "old_metadata, new_metadata, authz_data",
+    [
+        # standard resource_paths should get migrated into `authz` column with standard version number (0)
+        (
+            {"foo": "bar", "_resource_paths": ["/programs/DEV"]},
+            '{"foo": "bar"}',
+            '{"version": 0, "_resource_paths": ["/programs/DEV"]}',
+        ),
+        # default authz resource_paths do not get migrated into `data` column.
+        ({"foo": "bar"}, '{"foo": "bar"}', DEFAULT_AUTHZ_STR),
+        # handle null `data` columns (with populated `authz` columns)
+        (
+            {"_resource_paths": ["/programs/DEV"]},
+            None,
```

Comment:
how did you choose to use `None` instead of `{}`? i'm wondering which one we should use. `data` is not required in the DB but i'm not sure the API accepts a submission without it

Reply:
Should we skip the testing for `None`? Somehow I got my dev database to have rows with null metadata, so that was the motivation for including those tests. 
I'm adding tests for `{}` and those appear to work fine.
I think the migration is handling both `{}` and `None`.

Reply:
ok then i guess we can test both, just in case? afaik there should not be rows with None, but 🤷‍♀️